src/Day02.kt

```diff
@@ -1,15 +1,6 @@
-fun main() {
-    
-    fun remove(arr: IntArray, index: Int): IntArray {
-        if (index < 0 || index >= arr.size) {
-            return arr
-        }
-
-        val result = arr.toMutableList()
-        result.removeAt(index)
-        return result.toIntArray()
-    }
```

Comment:
You called it `remove` - so I removed it!  :D  

Really, it comes down to the fact that `getSmallestPerimeter` could be cut down significantly, so this went too!